action.yml

```
@@ -41,6 +45,14 @@ runs:
         else
           webotsTag=${{ inputs.webotsVersion }}
         fi
+        if [ -n '${{ inputs.webotsBaseUrl }}' ]; then
+          if [ -n '${{ inputs.webotsTag }}' ]; then
+            echo "WARNING: Ignoring webotsTag=${{ inputs.webotsTag }} because webotsBaseUrl was specified."
+          fi
+          webotsBaseUrl=${{ inputs.webotsBaseUrl }}
+        else
+          webotsBaseUrl='https://github.com/cyberbotics/webots/releases/download/${{ inputs.webotsTag }}'
```

Use the resolved tag to fix the CI
```suggestion
          webotsBaseUrl="https://github.com/cyberbotics/webots/releases/download/$webotsTag"
```

---

Fixed.